src/c_api/inspect.rs

```diff
@@ -153,16 +219,16 @@ fn convert_syms_list_to_c(syms_list: Vec<Vec<SymbolInfo>>) -> *const *const blaz
 ///
 /// The returned pointer should be free by [`blaze_syms_free`].
 #[no_mangle]
-pub unsafe extern "C" fn blaze_find_addrs(
-    symbolizer: *mut blazesym,
-    cfg: *const blazesym_sym_src_cfg,
+pub unsafe extern "C" fn blaze_lookup_elf(
```

Comment:
just "lookup" is a bit generic. If in the future we'll do other types of lookups we'll be running ourselves into corner with naming. Should this be a bit more specific, something like "blaze_lookup_syms_elf" or some variation of that?

Reply:
Sure, will change.

Reply:
How about `blaze_inspect_syms_elf`?

Reply:
yep, sounds good